tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Execute ansible deployment using qe-sap-deployment project.
+# https://github.com/SUSE/qe-sap-deployment
+
+use base 'sles4sap_publiccloud_basetest';
+use strict;
+use warnings;
+use testapi;
+use Mojo::File 'path';
+use publiccloud::utils;
+use sles4sap_publiccloud_lib;
+use qesapdeployment;
+use serial_terminal 'select_serial_terminal';
+
+sub test_flags {
+    return {fatal => 1, publiccloud_multi_module => 1};
+}
+
+sub run {
+    my ($self, $run_args) = @_;
+    select_serial_terminal;
+    my $ha_enabled = get_var("QESAP_HANA_HA_ENABLED") =~ /False|false|0/ ? 0 : 1;
```

Comment:
Is this new QESAP_ to be considered a generic qesapdeploy lib setting? Maybe not as far as it is in the test code and not in qesapdeployment?

If it is we can think about documenting it in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/0836e03e6d49b791d5986b597374992febe58d58/variables.md?plain=1#L244

If not, at least for Trento, I decided to add a prefix to them **HANASR_QESAP_HANA_HA_ENABLED**

Reply:
I will change it to HA_CLUSTER which is already standard variable for sles4sap/ha tests

Reply:
Fix applied